sssom/util.py

```diff
@@ -40,7 +40,8 @@
 MATCH_TYPE = "match_type"
 HUMAN_CURATED_MATCH_TYPE = "HumanCurated"
 
-_defining_features = [SUBJECT_ID, PREDICATE_ID, OBJECT_ID]
+# The 3 columns whose combination would be used as primary keys while merging/grouping
```

Comment:
```suggestion
#: The 3 columns whose combination would be used as primary keys while merging/grouping
```

the colon makes it a magical docstring